### PR TITLE
Fixes #17666: Node status API must not return an error when the node is not here

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/17666

### CODE REVIEW

File: webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala [hunk 1]
```diff
@@ -238,6 +238,10 @@ object NodeApi extends ApiModuleProvider[NodeApi] {
     val description = "List all accepted nodes with configurable details level"
     val (action, path)  = GET / "nodes"
   }
+  final case object GetNodesStatus extends NodeApi with ZeroParam with StartsAtVersion12 with SortIndex { val z = implicitly[Line].value
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala:241]: Should we increase the version here ?? 12 is version 6.1

**Comment by fanf** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala (hunk 1 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala]: done

**Review by VinceMacBuche — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**